lib/frontend_common/TaskclusterLogin.elm

```diff
@@ -56,6 +59,17 @@ update msg model =
         Logout ->
             ( model, taskclusterlogin_remove True )
 
+        CheckCertificate time ->
+            if isCertificateExpired time model then
+                ( Nothing, taskclusterlogin_remove True )
+            else
+                ( model, Cmd.none )
+
+
+getTimeAndThen : (Time -> Msg) -> Cmd Msg
```

Comment:
oh yes, i didn't see the init function. well if you think we need this function then we can leave it. otherwise your example looks good enough for me.